3ds2-example/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "payments-auth-integration-examples",
```

Comment:
change name to `adyen-node-online-payments-3ds2`

Reply:
✅